api/src/main/java/org/pmiops/workbench/db/model/Workspace.java

```diff
@@ -94,6 +97,10 @@ public boolean equals(Object obj) {
 
   private Set<WorkspaceUserRole> usersWithAccess = new HashSet<WorkspaceUserRole>();
 
+  public Workspace() {
+    setWorkspaceActiveStatusEnum(WorkspaceActiveStatus.ACTIVE);
```

Comment:
This looks a little scary to me; are you certain this works and won't just overwrite whatever status was in the database when we load a model from the database? (and then overwrite the database entry when we save it back?)

Reply:
it seems to work!  when I reload my workspaces after deleting, the deleted ones are still deleted